.travis.yml

```diff
@@ -17,11 +17,11 @@ jobs:
         - "docker"
       python: "3.8"
       install:
-        - python3 -m pip install cibuildwheel==1.3.0 twine virtualenv
+        - make jq.c
+        - python3 -m pip install cibuildwheel==1.3.0 twine virtualenv tox
       script:
+        - tox -e py  # TODO: verify if it works here
         - export CIBW_BEFORE_BUILD="pip install cython==0.29.16 && cython {project}/jq.pyx"
-        - export CIBW_TEST_REQUIRES="-r test-requirements.txt"
-        - export CIBW_TEST_COMMAND="nosetests {project}/tests"
```

Comment:
Why has this been removed? I think it's important to run the tests against the wheels to check that they're working correctly.